cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/config/ETLBatchConfig.java

```diff
@@ -29,34 +31,49 @@
 public final class ETLBatchConfig extends ETLConfig {
   private final String schedule;
   private final List<ETLStage> actions;
+  private final List<LocalizeResourceInfo> resourcesToLocalize;
```

Comment:
The app can creates localization directories for each plugin and have files inside symlink to the same localized file of the container.